modules/auxiliary/scanner/smb/smb_ms17_010.rb

```diff
@@ -56,6 +54,11 @@ def calculate_doublepulsar_xor_key(s)
     x & 0xffffffff  # this line was added just to truncate to 32 bits
```

Comment:
Probably not necessary because of how we split up the SMB signature, but it's good to keep because people are copying and pasting the shit out of this method.